ee/README.md

```diff
@@ -25,3 +25,15 @@ The [/ee](https://github.com/calendso/calendso/tree/main/ee) subfolder is the pl
 6. Open [Stripe Webhooks](https://dashboard.stripe.com/webhooks) and add `<CALENDSO URL>/api/integrations/stripepayment/webhook` as webhook for connected applications.
 7. Select all `payment_intent` events for the webhook.
 8. Copy the webhook secret (`whsec_...`) to `STRIPE_WEBHOOK_SECRET` in the .env file.
+
+## Setting up SAML login
+
+1. Set SAML_DATABASE_URL to a postgres database. Please use a different database than the main Cal instance since the migrations are separate for this database. For example `postgresql://postgres:@localhost:5450/cal-saml`
+2. Set SAML_ADMINS to a comma separated list of admin emails from where the SAML metadata can be uploaded and configured.
+3. Create a SAML application with your Identity Provider (IdP) using the instructions here - https://docs.google.com/document/d/
+   10J5tj5GduQB81MFHMmyrUr8tvEiHmp90ASgT5S6TG9s
```

Comment:
We shouldn't link instruction to a Google Doc IMO. Why won't we add a new file to `docs/saml-setup.md` and link to it?

Reply:
Will do